lib/index.js

```diff
@@ -44,5 +44,8 @@ History.prototype.lengthForward = require('./prototype/length-forward')
 History.prototype.backward = require('./prototype/backward')
 History.prototype.forward = require('./prototype/forward')
 History.prototype.move = require('./prototype/move')
+History.prototype.forgetAllInDirection = require('./prototype/forget-all-in-direction')
```

Comment:
This project [has documentation](https://github.com/PolicyStat/object-history).

The API part is from the code comments.

This new method, as I wrote, isn't there.